docs/Using Fleet/enroll-chromebooks.md

```diff
@@ -24,6 +24,22 @@ By default, the hostname for a Chromebook host will be blank. The hostname can b
 ## Debugging ChromeOS
 To learn how to debug the Fleetd Chrome extension, visit [here](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/Testing-and-local-development.md#fleetd-chrome-extension).
 
+## Potential issues and troubleshooting: 
+- 1 The extension does not install on our Chromebooks.
+- 2 Chrome web browsers on other OSs (Mac/Linux/Windows) get this extension (where it's not needed).
+
+Google Admin is arranged in a hierarchy of Organizational Units (OUs) tree. Each of the OUs can hold a combination of USERs and/or DEVICEs.
+Chrome extensions can be set for a specific OU (force-installed, allow-install or block). However, extensions can only be set at USERs level (not DEVICES).
+If a Chrome extension is deployed to an OU that only has DEVICES, it will not be installed. 
+On the other hand, if you deploy an extension to an OU that hold USERS with both ChromeBooks and
+managed Chrome web browsers (e.g. Chrome browser on a MacBook), it will deploy the extension to these Chrome web browsers.
+
+### Our recommendation: 
+- Create an OU that will hold all USERs with Chromebooks. Deploy our extension to that OU (Force-Install).
+- Create an OU that will hold all managed Chrome web browsers of the USERS above (Do not put the USERS in this OU, Just the managed Chrome web browsers). Make sure our extension is blocked on this OU. 
+
+> Note: When deployed on OSs other than ChromeOS, our Chrome extension will detect it and not perform any operation.  
```

Comment:
Hey @zwass I was chatting with @sharon-fdm about this. Wanted to clarify this line here. I think we would recommend to customers to do the configuration stated above as a first resort. 

But re: this note, if they just yolo and install it everywhere, is there a negative side effect? My understanding is that it would show up in the extensions page of Chrome, but not show up in the toolbar, which from a UI/UX perspective is fine. Is there a negative effect on the system from having these extensions around such as pings to the Fleet server, ghost hosts, something like that?

Reply:
We added a feature that will disable the functionality when loaded on a non-ChromeOS device. That means there won't be pings, ghost hosts, etc.